draft-ietf-rtcweb-jsep.xml

```diff
@@ -2344,9 +2344,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>The fields of the "o=" line MUST stay the
                         same except for the &lt;session-version&gt;
-                        field, which MUST increment if the session
-                        description changes in any way, including the
-                        addition of ICE candidates.</t>
+                        field, which MUST increment for each call to
+                        the createOffer() API. The only actions which
```

Comment:
I think actually:
Which MUST be incremented on each call to the createOffer method which would result in a different offer from the current local description. Implementations MAY
opt to increment the "o=" line on every call to createOffer.